pkg/virt-controller/watch/clone/clone.go

```diff
@@ -259,6 +260,24 @@ func (ctrl *VMCloneController) syncSourceVMTargetVM(key string, source *k6tv1.Vi
 		ctrl.logAndRecord(vmClone, TargetVMCreated, fmt.Sprintf("created target VM %s for clone %s", targetVMInfo.Name, vmClone.Name))
 		syncInfo.targetVMCreated = true
 
+		targetVM, ok := targetVMObj.(*k6tv1.VirtualMachine)
+		if !ok {
+			return addErrorToSyncInfo(syncInfo, fmt.Errorf("VM object from vm informer cannot be converted to *VirtualMachine: %v", err))
+		}
+
+		const vmNotReadyErrPattern = "target VM %s is still not ready - %s %s still in progress"
+		if targetVM.Status.SnapshotInProgress != nil {
```

Comment:
the snapshot for the clone is not on the target vm it was on the source vm, so seems redundant

Reply:
Yes, that's right. Thanks